autoload/lsp/ui/vim.vim

```diff
@@ -18,7 +18,7 @@ function! lsp#ui#vim#definition() abort
             \   'textDocument': lsp#get_text_document_identifier(),
             \   'position': lsp#get_position(),
             \ },
-            \ 'on_notification': function('s:handle_location', [l:server, s:last_req_id, 'definition']),
+            \ 'on_notification': function('s:handle_location', [1, l:ctx, l:server, 'definition']),
```

Comment:
`s:last_request_id` is actually very important due to async. Don't want slower responses to override newer response. Currently I make the last command always win.